basics/github.md

```diff
@@ -157,7 +160,7 @@ git push
 
 **Get thumbs**: once each reviewer is done with their review and happy with the code that will be merged, they should approve the PR and give a thumbs up  :+1:
 
-**Squash commits (optional)**: before merging, squash all your work into logical commits - or one single commit - that summarizes the changes.
+**Squash commits (optional)**: before merging, squash all your work into logical commits - or one single commit - that summarizes the changes. **Note**: try to avoid using squash commits to do the squashing!
```

Comment:
I feel like we have some confusing language going on. The difference between "squashing commits" and "squash commits" is unfortunate.

Maybe the Note could be something like "do not use GitHub's option to use squash commits to do the squashing"

Reply:
I totally agree. I will re-word this section a bit.